fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/events/pausefreespace.md

title: PAUSE/Freespace
images:
  - src: freespace_pause_image.jpg

>     alt: Tags hanging on chain-link fence with handwritten text about Poughkeepsie and feelings attached with red and blue cord

Reviewer: ```suggestion
    alt: Tags hanging on chain-link fence with text about Poughkeepsie and feelings attached with red and blue zip ties
```

  - src: dsc0004_2.jpg
    alt: Red and white metal soccer goal frame with net installed in grassy area with brick buildings in background
  - src: greta1.jpg

>     alt: Collection of colorful mutant vehicles and modified tricycles including pink and green converted vehicle with sidecar in urban lot

Reviewer: ```suggestion
    alt: Collection of colorful custom motorbikes including a green converted motorbike with sidecar in urban lot
```
